client/dashboard/app/style.scss

```diff
@@ -29,7 +29,7 @@ body {
 }
 
 a {
-	color: var( --dashboard__link-color );
+	color: var( --wp-admin-theme-color );
```

Comment:
I think this variable was in place to allow each app (.com, a4a) to customize it.

Reply:
They can customise `--wp-admin-theme-color`. Should it not always be the same color as the buttons etc.? Not sure why links are different

Reply:
Oh right they could be.